src/_image_wrapper.cpp

```diff
@@ -66,8 +66,8 @@ _get_transform_mesh(const py::object& transform, const py::ssize_t *dims)
 
     for (auto y = 0; y < dims[0]; ++y) {
         for (auto x = 0; x < dims[1]; ++x) {
-            *p++ = (double)x;
-            *p++ = (double)y;
+            *p++ = (double)x + 0.5;
+            *p++ = (double)y + 0.5;
```

Comment:
Could you leave a comment explaining what's happening? Or give these magic constants a variable name

Reply:
I added a comment that is hopefully helpful

Reply:
That helps, thanks!